lib/contentTypeParser.js

```diff
@@ -95,6 +95,8 @@ ContentTypeParser.prototype.existingParser = function (contentType) {
 }
 
 ContentTypeParser.prototype.getParser = function (contentType) {
+  contentType = contentType.trim()
```

Comment:
i feel this will slow us down

Reply:
From my point of view, `.trim` should place way lower place and it should be another `string`.
So, it would cache the original header and utilize the cache.

Reply:
I checked it and the Content-Type is matched correctly even if there are surrounding spaces - so trimming already happens somewhere

getParser is only called internally, as a result, I removed any behavioral changes from it, focusing on the public API